StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkStepUpVerification/NetworkingLinkStepUpVerificationViewController.swift

```diff
@@ -193,11 +193,7 @@ extension NetworkingLinkStepUpVerificationViewController: NetworkingOTPViewDeleg
     }
 
     func networkingOTPViewDidConfirmVerification(_ view: NetworkingOTPView) {
-        // we could call `showSmallLoadingView(false)` here but we do
-        // not because next step will be to transition to a different
-        // screen and its better to keep showing the loading to
-        // avoid the animation lag
-
+        showSmallLoadingView(true)
```

Comment:
yea you are 100% right haha - there is a restart

the fundamental reason is that the two parts "don't know about each other," so its easier to do a restart; the solutions to fix are:
1) See whether there's some way to restart the spinner only from the spot it was last on (...but this might be difficult and/or require upfront research/understanding)
2) Change the code and "connect" the two parts together (...but this _might_ trickle in some unfortunate code complexity)

for now, I am separating these two where the first order of business is to 'fix' the lack of a loading spinner (ex. user, on a slow internet connection, might be confused), and then the second order of business is polishing it a bit more (when other higher priority parts are done)